packages/cloud-cognitive/src/components/DataSpreadsheet/DataSpreadsheet.js

```diff
@@ -245,119 +327,141 @@ export let DataSpreadsheet = React.forwardRef(
         if ([35, 36, 37, 38, 39, 40].indexOf(keyCode) > -1) {
           event.preventDefault();
         }
-        // Clear out all cell selection areas if user uses any arrow key
+        // Clear out all cell selection areas if user uses any arrow key, except if the shift key is being held
         if ([37, 38, 39, 40].indexOf(keyCode) > -1) {
-          if (selectionAreas?.length) {
+          if (
+            selectionAreas?.length &&
+            keyCode !== 16 &&
+            !activeKeys.current.includes(16)
+          ) {
             setSelectionAreas([]);
             removeCellSelections();
           }
         }
-        switch (keyCode) {
-          // Tab
-          case 9: {
-            setSelectionAreas([]);
-            removeActiveCell();
-            setContainerHasFocus(false);
-            setActiveCellCoordinates(null);
-            break;
-          }
-          // Left
-          case 37: {
-            handleInitialArrowPress();
-            const coordinatesClone = { ...activeCellCoordinates };
-            if (coordinatesClone.column === 'header') {
-              return;
+        // Update list of activeKeys
+        if (!activeKeys.current?.includes(keyCode)) {
+          const activeClone = [...activeKeys.current];
+          activeKeys.current = [...activeClone, keyCode];
+        }
+        if (activeKeys.current?.length > 1) {
+          handleMultipleKeys();
+        }
+        // Allow arrow key navigation if there are less than two activeKeys OR
+        // if one of the activeCellCoordinates is in a header position
+        if (
+          !activeKeys.current.includes(16) ||
+          activeCellCoordinates.row === 'header' ||
+          activeCellCoordinates.column === 'header'
+        ) {
+          switch (keyCode) {
```

Comment:
No changes were made to this switch statement, I just wrapped it in the if statement above which made github highlight it as a bunch of changes

Reply:
Reviewer can set hide whitespace changes in settings in that case. 😄

Reply:
Cool! I did not know that